compiler/rustc_abi/src/lib.rs

```diff
@@ -11,7 +11,7 @@ use bitflags::bitflags;
 use rustc_data_structures::intern::Interned;
 #[cfg(feature = "nightly")]
 use rustc_data_structures::stable_hasher::StableOrd;
-use rustc_index::vec::{Idx, IndexSlice, IndexVec};
+use rustc_index::vec::{IndexSlice, IndexVec};
```

Comment:
Huh? How did this not issue dead code lint before? O_o

Reply:
The `layout.rs` file has `use super::*;` rather than doing its own imports.  So it's the change in the other file that lets this be removed.

Reply:
Ah, I see.